tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-self-use,invalid-name
+from typing import List
```

Comment:
You can also run `./scripts/pylint.sh` to run it on all of the code at once, instead of running it separately for `deep_qa/` and `tests/`.